### ISSUE TITLE
Validate `$appends` entries have corresponding accessors

### ISSUE THREAD

**Issue by alies-dev** [psalm/laravel-psalm-plugin#694]
When a model declares `$appends = ['full_name']`, Psalm should verify that a corresponding accessor exists (`getFullNameAttribute()` or a new-style `fullName()` Attribute accessor).

```php
class User extends Model {
    protected $appends = ['full_name', 'avatar_url'];
    // full_name has accessor — OK
    // avatar_url has no accessor — should warn
}
```

**Implementation:** `AfterClassLikeVisitInterface` — read `$appends`, check each entry against known accessors via the existing accessor detection logic in `ModelPropertyAccessorHandler`.
